covid19-etl/etl/ncbi.py

```diff
@@ -152,18 +194,20 @@ def submit_metadata(self):
 
         for node_name, _ in self.data_file.nodes.items():
             if node_name == "virus_sequence_run_taxonomy":
-                tasks.append(
-                    asyncio.ensure_future(
-                        self.files_to_virus_sequence_run_taxonomy_submission()
-                    )
-                )
+                continue
             else:
                 tasks.append(
                     asyncio.ensure_future(self.files_to_node_submissions(node_name))
                 )
 
         try:
-            loop.run_until_complete(asyncio.gather(*tasks))
+            results = loop.run_until_complete(asyncio.gather(*tasks))
+            loop.run_until_complete(
+                asyncio.gather(
+                    self.files_to_virus_sequence_run_taxonomy_submission(results[0])
```

Comment:
`results` stores the accession number list of `virus_sequence_contig`, `virus_sequence_peptide`, `virus_sequence_blastn`. Since they are identical, we can use any of them